src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -60,6 +62,11 @@
  * Type is {@code java.lang.String}.
  * Default value is {@code "."}.
  * </li>
+ * <li>
+ * Property {@code allowInlineReturn} - Specify if {@code {@return}} should be accepted.
```

Comment:
I think we can do validation without this property as behavior is backward compatible and restricted by javadoc tool.
checkstyle expects that content is passing javadoc tool without errors, same as checkstyle expects java code to be compilable first of all.

I updated original ticket with details.